cms/envs/common.py

```diff
@@ -815,6 +815,7 @@
     # other apps that are.  Django 1.8 wants to have imported models supported
     # by installed apps.
     'lms.djangoapps.verify_student',
+
```

Comment:
No, wait, this is for the CMS.  Then you might just as well revert the change to this file.  (Or just leave it as it is;  the empty line here doesn't hurt, it just seems pointless.)